x-pack/plugins/security_solution/public/entity_analytics/components/entity_details_flyout/tabs/risk_inputs.tsx

```diff
@@ -27,8 +30,30 @@ export interface AlertRawData {
   _id: string;
 }
 
-export const RiskInputsTab = ({ riskScore }: RiskInputsTabProps) => {
+const FIRST_RECORD_PAGINATION = {
+  cursorStart: 0,
+  querySize: 1,
+};
+
+export const RiskInputsTab = ({ entityType, entityName }: RiskInputsTabProps) => {
   const [selectedItems, setSelectedItems] = useState<AlertRawData[]>([]);
+  const nameFilterQuery = useMemo(() => {
+    if (entityType === RiskScoreEntity.host) {
+      return buildHostNamesFilter([entityName]);
+    } else if (entityType === RiskScoreEntity.user) {
+      return buildUserNamesFilter([entityName]);
+    }
+  }, [entityName, entityType]);
+
+  const { data: riskScoreData } = useRiskScore({
+    riskEntity: entityType,
+    filterQuery: nameFilterQuery,
+    onlyLatest: false,
+    pagination: FIRST_RECORD_PAGINATION,
+    skip: nameFilterQuery === undefined,
+  });
```

Comment:
Thanks for addressing this issue.

Quick question: Wouldn't this solution create an edge case for the following scenario?
1. Open entity flyout
2. Wait for a long time
3. Risk engine runs
4. Open risk input panel
5. The risk inputs panel snows data from the new run, and the entity panel shows data from the old run.

Reply:
Yes, I think so. 

But it's expected behaviour? I know that in some places we do have some auto-refresh, but not sure it's common pratice

Reply:
Hmm, I don't believe that it is expected. The panels will open side by side with inconsistent information. But I don't know if this edge case is relevant enough for us to worry about.